spec/models/bishop_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Bishop, type: :model do
+  describe '#move_legal?' do
+    let(:bishop){Bishop.create(row: 3, col: 3)}
+
+    it "is legal" do
+      expect(bishop.move_legal?(4,4)).to be true

```

Comment:
Can you change this to a further out square like 6,6? That way we can differentiate it from a king's legal move.